packages/ui/src/Layout/Layout.tsx

```diff
@@ -19,9 +32,34 @@ const Layout: React.FC<LayoutProps> = ({
   Notifications,
   children,
 }) => {
+  const { t } = useTranslation();
+
+  const adminsLink: Item[] = [
```

Comment:
The comment about hardcoding the values instead of passing them to layout applies here as well